.github/workflows/nightly-release.yml

```diff
@@ -148,12 +153,17 @@ jobs:
           echo "ASSETS_URL=$ASSETS_URL" >> $GITHUB_ENV
 
           echo "Upload assets"
-          cd distributions
+          pushd distributions
           for FILE in *zip; do
             [ -f "$FILE" ] || break
             echo "Upload asset $FILE"
             upload_zip_file $UPLOAD_URL $FILE ${{ secrets.GITHUB_TOKEN }}
           done
+          popd
+
+          pushd documentation/dynawoDocumentation
+          upload_pdf_file $UPLOAD_URL DynawoDocumentation.pdf ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We could also publish the tools example documentation?

Reply:
We won't be able to pull that off for this PR unfortunately.

Reply:
why is that?